python/lsst/drp/tasks/reprocess_visit_image.py

```diff
@@ -0,0 +1,581 @@
+    pipeBase.PipelineTaskConfig, pipelineConnections=ReprocessVisitImageConnections
+):
+    # To generate catalog ids consistently across subtasks.
+    id_generator = lsst.meas.base.DetectorVisitIdGeneratorConfig.make_field()
+
+    do_use_sky_corr = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Include the skyCorr input for background subtraction?",
+    )
+
+    snap_combine = pexConfig.ConfigurableField(
+        target=snapCombine.SnapCombineTask,
+        doc="Task to combine two snaps to make one exposure.",
+    )
+    repair = pexConfig.ConfigurableField(
+        target=repair.RepairTask,
+        doc="Task to repair cosmic rays on the exposure before PSF determination.",
+    )
+    detection = pexConfig.ConfigurableField(
+        target=lsst.meas.algorithms.SourceDetectionTask,
+        doc="Task to detect sources to return in the output catalog.",
+    )
+    sky_sources = pexConfig.ConfigurableField(
+        target=lsst.meas.algorithms.SkyObjectsTask,
+        doc="Task to generate sky sources ('empty' regions where there are no detections).",
+    )
+    deblend = pexConfig.ConfigurableField(
+        target=lsst.meas.deblender.SourceDeblendTask, doc="Split blended sources into their components."
+    )
+    measurement = pexConfig.ConfigurableField(
+        target=lsst.meas.base.SingleFrameMeasurementTask,
+        doc="Task to measure sources to return in the output catalog.",
+    )
+    normalized_calibration_flux = pexConfig.ConfigurableField(
+        target=lsst.meas.algorithms.NormalizedCalibrationFluxTask,
+        doc="Task to normalize the calibration flux (e.g. compensated tophats).",
+    )
+    apply_aperture_correction = pexConfig.ConfigurableField(
+        target=lsst.meas.base.ApplyApCorrTask,
+        doc="Task to apply aperture corrections to the measured sources.",
+    )
+    set_primary_flags = pexConfig.ConfigurableField(
+        target=lsst.meas.algorithms.setPrimaryFlags.SetPrimaryFlagsTask,
+        doc="Task to add isPrimary to the catalog.",
+    )
+    catalog_calculation = pexConfig.ConfigurableField(
+        target=lsst.meas.base.CatalogCalculationTask,
+        doc="Task to compute catalog values using only the catalog entries.",
+    )
+    post_calculations = pexConfig.ConfigurableField(
+        target=lsst.meas.base.SingleFrameMeasurementTask,
+        doc="Task to compute catalog values after all other calculations have been done.",
+    )
+    calib_match_radius = pexConfig.Field(
+        dtype=float,
+        default=0.2,
+        doc="Radius in arcseconds to cross-match calib_sources to the output catalog.",
+    )
+
+    def setDefaults(self):
+        super().setDefaults()
+
+        # No need to redo background: we have the global background model.
+        self.detection.reEstimateBackground = False
+        self.detection.doTempLocalBackground = False
+
+        # TODO: Note: these apertures were selected for HSC, and may not be
```

Comment:
Is there a ticket for this?

Reply:
No: this has been an oddity forever. Do you think we should pay attention to it, or just forget about it since every instrument except LATISS has been using these same values?

Reply:
I would just change TODO to NOTE for reference.